scanner/src/funTest/kotlin/scanners/AbstractPathScannerWrapperFunTest.kt

```diff
@@ -33,6 +33,7 @@ import org.ossreviewtoolkit.model.LicenseFinding
 import org.ossreviewtoolkit.model.PackageType
 import org.ossreviewtoolkit.model.config.DownloaderConfiguration
 import org.ossreviewtoolkit.model.config.FileArchiverConfiguration
+import org.ossreviewtoolkit.model.config.RepositoryConfiguration
```

Comment:
I'm against making the scanner consume the excludes.
One reason is consistency, the other is that adjusting the excludes would invalidate the scan results. In order to apply the changed excludes you would need to re-scan. This is IMO an unnecessary waste of human time. 

If such findings would be flagged by a policy rule, then consumption of excludes would be consistent with license findings and copyright findings handling.

Reply:
Point taken.
@sschuberth : Care to share your opinion ? This was originally your idea to generate FossID exclusions rules from the ORT exclusion rules.

For some context, I must say this is something we badly need, to make each FossID scan more independent (the current logic of copying the rules from last completed scan being too error prone).

Reply:
@fviernau This would also help for https://github.com/oss-review-toolkit/ort/issues/5018.

Reply:
As long as excludes are only used by scanners for which the scan results are not stored, this would not be a problem. This is currently the case for all implementations of `ScannerWrapper` that set `override val criteria: ScannerCriteria? = null`, like `FossId`.
One way to ensure that `ScannerWrapper`s with `criteria != null` do not have access to the excludes would be to filter the `ScanContext` before passing it to such scanners.

Reply:
I like @mnonnenmacher  idea. The reason why I was afraid of consuming excludes was complexity, along with the need to make the scanner scale better some day. The more complicated we make things here, the more afraid developers will become to touch it, the less likely we will have better scaling in the scanner. Martin's proposal addresses this, because for scanners which do not store anything, there is no performance speed up from caching anyway.

Reply:
Thanks a lot @mnonnenmacher  for your idea !